packages/website/.eslintrc.js

```diff
@@ -19,6 +19,8 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaVersion: 9, // Allows for the parsing of modern ECMAScript features
+    project: './tsconfig.json',
+    tsconfigRootDir: __dirname,
```

Comment:
helps with eslint-vscode weirdness that can be flakey. https://github.com/typescript-eslint/typescript-eslint/issues/1681#issuecomment-595327139